ClientTests/EcosiaBookmarkTests.swift

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+@testable import Client
+@testable import Core
+import Shared
+import Storage
+
+final class EcosiaBookmarkTests: ProfileTest {
+//    func testEcosiaImportBookmarks() {
+//        try? FileManager.default.removeItem(at: FileManager.pages)
+//        Core.User.shared.migrated = false
+//
+//        let urls = [URL(string: "https://ecosia.org")!,
+//                    URL(string: "https://guacamole.com")!,
+//                    URL(string: "schemeless.com")!,
+//                    URL(string: "error://")!]
+//
+//        Core.Favourites().items = urls.map {
+//            .init(url: $0, title: "")
+//        }
+//        let expect = expectation(description: "")
+//
+//        PageStore.queue.async {
+//            DispatchQueue.main.async {
+//                self.withTestProfile { profile -> Void in
+//                    let ecosia = EcosiaImport(profile: profile, tabManager: .init(profile: profile, imageStore: nil))
+//                    ecosia.migrate { _ in
+//                        DispatchQueue.main.async {
+//                            profile.places.getBookmarksTree(rootGUID: "mobile______", recursive: true) >>== { folder in
+//                                guard let folder = folder as? BookmarkFolder else { return }
+//                                XCTAssertEqual(3, folder.children!.count)
+//                                try? FileManager.default.removeItem(at: FileManager.favourites)
+//                                expect.fulfill()
+//                            }
+//                            
+//                            _ = profile.places.forceClose()
+//                            try? profile.files.remove("profile-test_places.db")
+//                        }
+//                    }
+//                }
+//            }
+//        }
+//        waitForExpectations(timeout: 20)
```

Comment:
yeah testing this is tricky.. what you could do is leave the code uncommented but just deactivate this Test in the scheme. That way we can still it run it locally in XCode without the need to change the code :)

Reply:
..and it's not run on CI then

Reply:
That's a good idea